drivers/cc110x/include/cc110x_settings.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
 /**
  * @brief   Configuration register values for CC1100/CC1101 transceivers
  */
-extern const char cc110x_conf[CC110X_CONF_SIZE];
+extern const unsigned char cc110x_conf[CC110X_CONF_SIZE];
```

Comment:
```suggestion
extern const uint8_t cc110x_conf[CC110X_CONF_SIZE];
```

Let's go for `uint8_t`, to keep it consistent.